frontend/src/features/projects/dispose/components/ApprovalActions.tsx

```diff
@@ -30,6 +30,11 @@ export const ApprovalActions = ({
   const { values, submitForm, validateForm } = useFormikContext<any>();
   const [cancel, setCancel] = useState(false);
   const { noFetchingProjectRequests } = useStepForm();
+  useEffect(() => {
```

Comment:
See my comment at the end of this review. This ensures that the value of submitStatusCode is set correctly before submitForm() is called.